templates/iam_policy/custodial_copy_policy.json.tpl

```diff
@@ -38,15 +38,15 @@
       ],
       "Effect": "Allow",
       "Resource": [
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery:*:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-webapp:*:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-webapp:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-webapp",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-builder:*:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-builder:*",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery-builder"
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/custodial-copy:*:*",
```

Comment:
1. Maybe `custodial-copy-backend`?
2. Do you think it's worth using variables instead?

Reply:
1. Ok
2. It doesn't really save many characters so I'll leave it I think.